packages/server/lib/plugins/index.coffee

```diff
@@ -37,7 +37,7 @@ module.exports = {
 
       registeredEvents = {}
 
-      pluginsProcess = cp.fork(path.join(__dirname, "child", "index.js"), ["--file", config.pluginsFile])
+      pluginsProcess = cp.fork(path.join(__dirname, "child", "index.js"), ["--file", config.pluginsFile], { stdio: "inherit" })
```

Comment:
So we did forget to do this... on regular plugins?

AKA users wouldn't see `console.log` messages from node in `stdout` right?

If this is the case, please open a separate issue describing this. I will submit a PR against the new issue and remove this from here so it can be tracked separately.

Reply:
@chrisbreiding ^^

Reply:
I just tested this on `develop` and `console.log` makes it to `stdout`, so I think this is superfluous. In other words, it works without `{ stdio: "inherit" }`.